packages/optimizers/sharp/src/SharpOptimizer.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import {Optimizer} from '@parcel/plugin';
+import {blobToBuffer} from '@parcel/utils';
+import path from 'path';
+import sharp from 'sharp';
+
+const OPTIONS = {
+  jpeg: {
+    quality: 75,
```

Comment:
To match https://github.com/GoogleChromeLabs/squoosh/blob/2bf0b904cd3ebf831c9b47e6f59ab508ae968f81/src/features/encoders/mozJPEG/shared/meta.ts#L23 (rather than proposing it as a new default).